sites/orgdocs/docs/designs/bibi/instructions/readme.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: 'Bibi Body Block: Sewing Instructions'
+sidebar_label: Sewing Instructions
+sidebar_position: 20
+---
+
+:::note
+
+###### Bibi is a customizable pattern building block
+
+Bibi can be sewn as a simple top, but it is mostly designed as a building block for other patterns or custom designs.
+
+Bibi is highly customizable, but it does not in itself represent a specific garment or style.
+Some option combinations might not make much sense, and you may need to experiment with different options.
+
+You may want to finish the hem, armholes, and/or neck using knit bands, knit binding or ribbing fabric.
+These instructions give basic guidelines for that,
+but you might also want to look at instructions from other designs, like Sven, Aaron or Teagan.
+
+There is also lots of other stuff you could customize.
+
+:::
+
+:::note
+
+As with all knits and stretch fabrics, a serger/overlock will make your life easier.
+If you don't have one, don't despair. You don't really need one.
+All serged seams on Bibi can also be sewn with a short, narrow zigzag stitch (~2 mm wide) on a standard sewing machine.
+
+For the topstitching steps, a coverlock works best, but if you don't have a coverlock, a twin needle will also give good results.
+Of course, you can also use a zigzag stitch.
+
+:::
+
+### Step 0: Prepare the fabric
```

Comment:
The heading level is off in this file. The top-level headings are H3, but they should be H2 (`##` instead of `###`)

Reply:
Fixed